jenkins/Jenkinsfile-publish

```diff
@@ -0,0 +1,34 @@
+@Library(['sym-pipeline']) _
+
+import com.symphony.cicd.SymphonyCICDUtils
+
+cicdUtils = new SymphonyCICDUtils()
```

Comment:
I guess we could get rid of that variable too as it does not seem to be used

Reply:
And it is an internal library, better if we avoid to share it on public repos